src/renderers/dom/client/eventPlugins/SimpleEventPlugin.js

```diff
@@ -30,451 +30,104 @@ var emptyFunction = require('emptyFunction');
-    phasedRegistrationNames: {
-      bubbled: 'onDrag',
-      captured: 'onDragCapture',
-    },
-  },
-  dragEnd: {
-    phasedRegistrationNames: {
-      bubbled: 'onDragEnd',
-      captured: 'onDragEndCapture',
-    },
-  },
-  dragEnter: {
-    phasedRegistrationNames: {
-      bubbled: 'onDragEnter',
-      captured: 'onDragEnterCapture',
-    },
-  },
-  dragExit: {
-    phasedRegistrationNames: {
-      bubbled: 'onDragExit',
-      captured: 'onDragExitCapture',
-    },
-  },
-  dragLeave: {
-    phasedRegistrationNames: {
-      bubbled: 'onDragLeave',
-      captured: 'onDragLeaveCapture',
-    },
-  },
-  dragOver: {
-    phasedRegistrationNames: {
-      bubbled: 'onDragOver',
-      captured: 'onDragOverCapture',
-    },
-  },
-  dragStart: {
-    phasedRegistrationNames: {
-      bubbled: 'onDragStart',
-      captured: 'onDragStartCapture',
-    },
-  },
-  drop: {
-    phasedRegistrationNames: {
-      bubbled: 'onDrop',
-      captured: 'onDropCapture',
-    },
-  },
-  durationChange: {
-    phasedRegistrationNames: {
-      bubbled: 'onDurationChange',
-      captured: 'onDurationChangeCapture',
-    },
-  },
-  emptied: {
-    phasedRegistrationNames: {
-      bubbled: 'onEmptied',
-      captured: 'onEmptiedCapture',
-    },
-  },
-  encrypted: {
-    phasedRegistrationNames: {
-      bubbled: 'onEncrypted',
-      captured: 'onEncryptedCapture',
-    },
-  },
-  ended: {
-    phasedRegistrationNames: {
-      bubbled: 'onEnded',
-      captured: 'onEndedCapture',
-    },
-  },
-  error: {
-    phasedRegistrationNames: {
-      bubbled: 'onError',
-      captured: 'onErrorCapture',
-    },
-  },
-  focus: {
-    phasedRegistrationNames: {
-      bubbled: 'onFocus',
-      captured: 'onFocusCapture',
-    },
-  },
-  input: {
-    phasedRegistrationNames: {
-      bubbled: 'onInput',
-      captured: 'onInputCapture',
-    },
-  },
-  invalid: {
-    phasedRegistrationNames: {
-      bubbled: 'onInvalid',
-      captured: 'onInvalidCapture',
-    },
-  },
-  keyDown: {
-    phasedRegistrationNames: {
-      bubbled: 'onKeyDown',
-      captured: 'onKeyDownCapture',
-    },
-  },
-  keyPress: {
-    phasedRegistrationNames: {
-      bubbled: 'onKeyPress',
-      captured: 'onKeyPressCapture',
-    },
-  },
-  keyUp: {
-    phasedRegistrationNames: {
-      bubbled: 'onKeyUp',
-      captured: 'onKeyUpCapture',
-    },
-  },
-  load: {
-    phasedRegistrationNames: {
-      bubbled: 'onLoad',
-      captured: 'onLoadCapture',
-    },
-  },
-  loadedData: {
-    phasedRegistrationNames: {
-      bubbled: 'onLoadedData',
-      captured: 'onLoadedDataCapture',
-    },
-  },
-  loadedMetadata: {
-    phasedRegistrationNames: {
-      bubbled: 'onLoadedMetadata',
-      captured: 'onLoadedMetadataCapture',
-    },
-  },
-  loadStart: {
-    phasedRegistrationNames: {
-      bubbled: 'onLoadStart',
-      captured: 'onLoadStartCapture',
-    },
-  },
-  // Note: We do not allow listening to mouseOver events. Instead, use the
```

Comment:
Why was this comment here? I don’t understand what it refers to because `mouseOver` is defined just below. Is this still relevant and should some version of this be copied to new code?